x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -500,6 +515,119 @@ export class AlertsClient {
     }
   }
 
+  public async getAlertSummary({
+    gte,
+    lte,
+    featureIds,
+    filter,
+    fixedInterval = '1m',
+    index,
+  }: GetAlertSummaryParams) {
+    try {
+      let indexToUse = index;
+      if (featureIds && !isEmpty(featureIds)) {
+        indexToUse = (
+          (await this.getAuthorizedAlertsIndices(
+            Array.isArray(featureIds) ? featureIds : [featureIds]
+          )) ?? []
+        ).join(',');
+      }
+
+      // first search for the alert by id, then use the alert info to check if user has access to it
+      const [activeResponse, recoveredResponse] = await Promise.all([
+        this.singleSearchAfterAndAudit({
+          index: indexToUse,
+          operation: ReadOperations.Get,
+          aggs: {
+            active_alerts_bucket: {
+              date_histogram: {
+                field: ALERT_TIME_RANGE,
+                fixed_interval: fixedInterval,
+                hard_bounds: {
+                  min: gte,
+                  max: lte,
+                },
+                extended_bounds: {
```

Comment:
do we need to specify `min_doc_count: 0` to ensure empty buckets show up with the `extended_bounds`? 

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html#search-aggregations-bucket-histogram-aggregation-extended-bounds

Reply:
That would be a good idea so when we draw empty charts (and forget to add bounds to the axes) it will have the correct time buckets.